src/order/models.py

```diff
@@ -154,7 +154,9 @@ def auto_create_orders(self, delivery_date, clients):
             try:
                 # If an order is already created, skip order items creation
                 # (if want to replace, must be deleted first)
-                Order.objects.get(client=client, delivery_date=delivery_date)
+                order = Order.objects.get(client=client,
+                                          delivery_date=delivery_date)
+                orders.append(order)
```

Comment:
Nice trick! This way, we can not replace completly replace the table containing the orders.